ur_description/config/ur10e/visual_parameters.yaml

```diff
@@ -36,7 +36,7 @@ mesh_files:
         name: "LightGrey"
         color: "0.7 0.7 0.7 1.0"
     collision:
-      mesh: "package://ur_description/meshes/ur10e/collision/forearm.stl"
+      mesh: "package://ur_description/meshes/ur10e/visual/forearm.stl"
```

Comment:
I'm tempted to use the visual for all links...
and also for all robot variants...
...for consistency reasons - we would just run into a similar issue with another link some time and start debugging this all over again

Reply:
Yeah, was thinking the same. But I'm also not so comfy with needing this really. I'd rather keep some distance.

Reply:
well, we havent used any explicit padding so far
and now we get planning failures for one of the only link that actzally has a larger collision mesh...and instantly we have no clue anymore whats going on...